common/core_api_review/kibana.api.ts

```diff
@@ -0,0 +1,72 @@
+// @public

```

Comment:
I'm sorta conflicted on the choice to put this in the "common" directory at the root of the repository. I'm not sure why this directory exists in the first place, but could you help me understand why this placement makes sense?

I'm conflicted because I can't really think of a _better_ place to put it, though I feel like we might be able to come up with something a little more self explanatory than "common".

Reply:
Yeah I don't really like "common" either, it doesn't mean anything to me or describe to me what's inside of this folder. The `common` directory is the convention used by api-generator and Microsoft uses that convention in most of their projects. The graphql introspection files seem to follow the same convention, so I thought I'll stick to that in the absence of any better ideas.

We don't want to put this in `/src` cause it is a generated file, not a source file. We don't want it in `/build` cause it needs to get commited so that we can compare changes to the api signature. We could potentially put this is `/dev` or create a more descriptive top-level directory like `api_signature` which could also be the home for the GraphQL introspection files.

Reply:
If it helps, we have one precedent in the repo today for committing a generated file, and that is under a `dist` directory at the root of that package: https://github.com/elastic/kibana/tree/master/packages/kbn-pm/dist

Reply:
api-extractor now creates a .md file so it's safe to put the API review file with the source code, so I've moved the review file into the /public /server directories.